OUDS/Core/Tokens/RawTokens/Sources/Values/FontRawTokens+Values.swift

```diff
@@ -69,48 +66,17 @@ extension FontRawTokens {
     public static let fontSize1250: FontSizeRawToken = 56
     public static let fontSize1450: FontSizeRawToken = 64
     public static let fontSize1850: FontSizeRawToken = 72
-
-    // MARK: Primitive token - Typography - Line height
-
-    // WARNING: values in pixels and not in points, do not use as is!
-
-    public static let fontLineHeight250: FontLineHeightRawToken = 16
-    public static let fontLineHeight350: FontLineHeightRawToken = 20
-    public static let fontLineHeight450: FontLineHeightRawToken = 24
-    public static let fontLineHeight550: FontLineHeightRawToken = 28
-    public static let fontLineHeight650: FontLineHeightRawToken = 32
-    public static let fontLineHeight750: FontLineHeightRawToken = 36
-    public static let fontLineHeight850: FontLineHeightRawToken = 40
-    public static let fontLineHeight950: FontLineHeightRawToken = 44
-    public static let fontLineHeight1050: FontLineHeightRawToken = 48
-    public static let fontLineHeight1150: FontLineHeightRawToken = 52
-    public static let fontLineHeight1250: FontLineHeightRawToken = 56
-    public static let fontLineHeight1350: FontLineHeightRawToken = 60
-    public static let fontLineHeight1450: FontLineHeightRawToken = 64
-    public static let fontLineHeight1850: FontLineHeightRawToken = 72
-    public static let fontLineHeight2050: FontLineHeightRawToken = 80
-
-    // MARK: Primitive token - Typography - Letter spacing
-
-    // WARNING: values in pixels and not in points, do not use as is!
-
-    public static let fontLetterSpacing150: FontLetterSpacingRawToken = 150
-    public static let fontLetterSpacing175: FontLetterSpacingRawToken = 175
-    public static let fontLetterSpacing200: FontLetterSpacingRawToken = 200
-    public static let fontLetterSpacing250: FontLetterSpacingRawToken = 250
-    public static let fontLetterSpacing300: FontLetterSpacingRawToken = 300
-    public static let fontLetterSpacing350: FontLetterSpacingRawToken = 350
-    public static let fontLetterSpacing450: FontLetterSpacingRawToken = 450
-    public static let fontLetterSpacing550: FontLetterSpacingRawToken = 550
-    public static let fontLetterSpacing650: FontLetterSpacingRawToken = 650
-    public static let fontLetterSpacing750: FontLetterSpacingRawToken = 750
-    public static let fontLetterSpacing850: FontLetterSpacingRawToken = 850
-    public static let fontLetterSpacing950: FontLetterSpacingRawToken = 950
-    public static let fontLetterSpacing1050: FontLetterSpacingRawToken = 1_050
-    public static let fontLetterSpacing1150: FontLetterSpacingRawToken = 1_150
-    public static let fontLetterSpacing1250: FontLetterSpacingRawToken = 1_250
-    public static let fontLetterSpacing1450: FontLetterSpacingRawToken = 1_450
-    public static let fontLetterSpacing1850: FontLetterSpacingRawToken = 1_850
```

Comment:
This PR has some mistakes in it, so I'll fix it in a new one.

However, I can see that you have letter spacings like: `150` to `1_850`.
The values I have by default are rather `0.18` to `-1.08`.
What should I do for them?

Reply:
You should keep the values you got, i.e. from 0.18 to -1.08.
The value you saw before the update were the values picked I don't remeber where, but I can say they are not usable at all with _SwiftUI_ API :)

I made a test using _Xcode_ with the _Preview_, the rendering is far more nice and interesting with _your_ values, not the old ones.

For the source code bellow (`tracking()`) is the API to use for letter spacing
```swift
#Preview {

    Text("(0.18) Tracking - Nous vivons tous dans le ventre d'un chien géant")
        .tracking(0.18)
        .border(.green)

    Text("(0.18) Kerning - Nous vivons tous dans le ventre d'un chien géant")
        .kerning(0.18)
        .border(.green)

    Text("(-1.08) Tracking - Nous vivons tous dans le ventre d'un chien géant")
        .tracking(-1.08)
        .border(.blue)

    Text("(-1.08) Kerning - Nous vivons tous dans le ventre d'un chien géant")
        .kerning(-1.08)
        .border(.blue)
    
    Text("(10) Tracking - Nous vivons tous dans le ventre d'un chien géant")
        .tracking(10)
        .border(.red)

    Text("(10) Kerning - Nous vivons tous dans le ventre d'un chien géant")
        .kerning(10)
        .border(.red)
}
```

I have the following result
![tracking and kerning](https://github.com/user-attachments/assets/4144f1ee-9530-46a3-93dd-9c6dc196cfc5)

Note we have two API for letter spacing, [tracking](https://developer.apple.com/documentation/swiftui/view/tracking(_:)) and [kerning](https://developer.apple.com/documentation/swiftui/view/kerning(_:)), with small differences, I will check with @MaximeTonnerre  which one is the most suitable.

In few words, _your_ values are the usable ones, not the old ones :)